lib/components/molecules/Form/DragDropInput/DragDropInput.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
+import { Field } from 'formik';
+import styled from 'styled-components';
+
+import Label from '../../../atoms/Label';
+import Input from '../../../atoms/Input';
+import styles from './DragDropInput.style';
+import useOpenFileDialog from '../../../../hooks/file-upload/useOpenFileDialog';
+import useDragandDrop from '../../../../hooks/file-upload/useDragandDrop';
```

Comment:
I think the structure you have used for storing these hooks could be improved. useDragandDrop Hooks are not only related to file upload. They can be used with other Drag and Drop features.